tornado/db.py

```diff
@@ -71,6 +71,15 @@ class DatasetFile(BaseModel):
     class Meta:
         db_table = 'dataset_file'
 
+class DatasetLogo(BaseModel):
+    dataset_logo = PrimaryKeyField(db_column='dataset_logo_pk')
+    dataset      = ForeignKeyField(db_column='dataset_pk', rel_model=Dataset, to_field='dataset')
+    mimetype     = CharField()
```

Comment:
Both `mimetype` and `data` are allowed to be `NULL`, so `(null=True)` for these.